editor/export/editor_export_platform_pc.cpp

```diff
@@ -151,15 +151,27 @@ Error EditorExportPlatformPC::prepare_template(const Ref<EditorExportPreset> &p_
 		return ERR_FILE_NOT_FOUND;
 	}
 
-	String wrapper_template_path = template_path.get_basename() + "_console.exe";
+	// Matching the extensions in platform/windows/console_wrapper_windows.cpp
+	static const char *const wrapper_extensions[] = {
+		".console.exe",
+		"_console.exe",
+		" console.exe",
+		"console.exe",
+		nullptr,
+	};
 	int con_wrapper_mode = p_preset->get("debug/export_console_wrapper");
 	bool copy_wrapper = (con_wrapper_mode == 1 && p_debug) || (con_wrapper_mode == 2);
 
 	Ref<DirAccess> da = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
 	da->make_dir_recursive(p_path.get_base_dir());
 	Error err = da->copy(template_path, p_path, get_chmod_flags());
-	if (err == OK && copy_wrapper && FileAccess::exists(wrapper_template_path)) {
-		err = da->copy(wrapper_template_path, p_path.get_basename() + ".console.exe", get_chmod_flags());
+	if (err == OK && copy_wrapper) {
+		for (int i = 0; wrapper_extensions[i]; ++i) {
+			if (FileAccess::exists(template_path.get_basename() + wrapper_extensions[i])) {
+				err = da->copy(template_path.get_basename() + wrapper_extensions[i], p_path.get_basename() + ".console.exe", get_chmod_flags());
```

Comment:
```suggestion
			const String wrapper_path = template_path.get_basename() + wrapper_extensions[i];
			if (FileAccess::exists(wrapper_path)) {
				err = da->copy(wrapper_path, p_path.get_basename() + ".console.exe", get_chmod_flags());
```
Will push once CI has finished